protocols/tests/test_migration/test_migration_reports_4_0_0_to_reports_3_0_0.py

```diff
@@ -0,0 +1,70 @@
+import factory.fuzzy
+from random import randint
+
+from protocols import reports_4_0_0
+from protocols import reports_3_0_0
+from protocols.util.dependency_manager import VERSION_61
+from protocols.util.dependency_manager import VERSION_500
+from protocols.util.dependency_manager import VERSION_400
+from protocols.util.factories.avro_factory import FactoryAvro
+from protocols.util.factories.avro_factory import GenericFactoryAvro
+from protocols.tests.test_migration.base_test_migration import TestCaseMigration
+from protocols.migration import MigrateReports400To300
+
+
+class ActionFactory(FactoryAvro):
+    class Meta:
+        model = reports_4_0_0.Actions
+
+    _version = VERSION_400
+
+    actionType = factory.fuzzy.FuzzyChoice(['therapy', 'therapeutic', 'prognosis', 'diagnosis'])
+    status = factory.fuzzy.FuzzyChoice(['clinical', 'pre-clinical'])
+    evidence = ["this", "that"]
+    drug = factory.fuzzy.FuzzyText()
+    variantActionable = factory.fuzzy.FuzzyChoice([True, False])
+    comments = ["this", "that"]
+    url = factory.fuzzy.FuzzyText()
+    evidenceType = factory.fuzzy.FuzzyText()
+    source = factory.fuzzy.FuzzyText()
+
+
+class TestMigrateReports5To400(TestCaseMigration):
+
+    old_model = reports_4_0_0
+    new_model = reports_3_0_0
+
+    def setUp(self):
```

Comment:
This can be removed. Only necessary to create mocked cancer report events in reports 4.0.0

Reply:
:+1: